packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/components/TraitGridFilters.tsx

```diff
@@ -405,17 +409,28 @@ const mapDispatchToProps = (dispatch) => ({
 const connector = connect(null, mapDispatchToProps)
 
 type OwnProps = {
-  activeTab?: 'ITEMS' | 'EVENTS' | 'EXPLORE'
+  activeTab?: 'ITEMS' | 'ACTIVITY' | 'EXPLORE'
   collections: OwnerQuery['assets'][0]['collection'][]
   formActions: typeof actions.form
   formValues: NftFilterFormValuesType
   numOfResults?: number
-  setActiveTab: React.Dispatch<React.SetStateAction<'ITEMS' | 'EVENTS'>>
+  setActiveTab: React.Dispatch<React.SetStateAction<'ITEMS' | 'ACTIVITY'>>
   setIsFilterOpen: React.Dispatch<React.SetStateAction<boolean>>
   setRefreshTrigger: React.Dispatch<React.SetStateAction<number>>
   showSortBy?: boolean
-  tabs: Array<'ITEMS' | 'EVENTS' | 'EXPLORE'>
+  tabs: Array<'ITEMS' | 'ACTIVITY' | 'EXPLORE'>
 }
+
+const enhance = compose<any>(
+  reduxForm({
+    form: 'nftFilter',
+    initialValues: {
+      sortBy: `${AssetSortFields.ListingDate}-DESC`
+    }
+  }),
+  connector
```

Comment:
You don't need to attach reduxForm here if the only thing you want to do is reset the sort by field

```
formActions.change('nftFilter', 'sortBy', recentlyListed)
```

doesnt need the connected form

Reply:
How do I remove it ?